public/app/core/services/backend_srv.ts

```diff
@@ -212,6 +213,14 @@ export class BackendSrv implements BackendService {
               const firstAttempt = i === 0 && options.retry === 0;
 
               if (error.status === 401 && isLocalUrl(options.url) && firstAttempt && isSignedIn) {
+                if (error.data.message.includes('revoked')) {
+                  appEvents.emit(CoreEvents.showModalReact, {
+                    component: TokenRevokedModal,
+                  });
+
+                  return of(null);
```

Comment:
If you want to abort the retry without throwing an error, I think you should use EMPTY instead:
```suggestion
                  return  EMPTY;
```